README.md

```diff
@@ -64,7 +64,7 @@ LinkedIn: [linkedin.com/in/nicorithner/](https://www.linkedin.com/in/nicorithner
 
 ### Extension Goals for Future Iterations
 
-- A User can create a menu indicating preferred ingredients
+- An user can create a menu indicating preferred ingredients
```

Comment:
Hey Nico, 

Grammatically,  we should use "A user " instead of "An user", can we please correct all of them in the readME file? 

Thank you!